store/store.go

```diff
@@ -77,27 +53,17 @@ func (s *GDStore) InitPrefix(p string) error {
 	if _, e := s.Get(p); e != nil {
 		switch e {
 		case store.ErrKeyNotFound:
-			log.WithField("prefix", p).Debug("prefix not found, initing")
-
 			if e := s.Put(p, nil, &store.WriteOptions{IsDir: true}); e != nil {
-				log.WithFields(log.Fields{
-					"preifx": p,
-					"error":  e,
-				}).Error("error initing prefix")
-
 				return e
+			} else {
+				log.WithField("prefix", p).Debug("Prefix not found. Created prefix.")
```

Comment:
Split this log. Log "Prefix not found" when you enter the case, and "Created prefix" only on success.